src/onnx/parse_convolution.cpp

```diff
@@ -42,18 +43,153 @@ struct parse_convolution : op_parser<parse_convolution>
+        }
+    }
+
+    static float get_symmetric_value(const instruction_ref& input)
+    {
+        float symmetric_value = 0;
+        // adjust symmetric zero point value for uint8 types
+        if(input->get_shape().type() == migraphx::shape::uint8_type)
+        {
+            symmetric_value = 128;
+        }
+        return symmetric_value;
+    }
+
+    static instruction_ref get_zero_point(const instruction_ref& input,
+                                          int index,
+                                          const bool is_quant_conv,
+                                          onnx_parser::node_info& info,
+                                          const std::vector<instruction_ref>& args)
+    {
+        instruction_ref ret = input;
+        if(args.size() > index)
+        {
+            // Check for type mismatch on parse
+            if(input->get_shape().type() != args[index]->get_shape().type())
+                MIGRAPHX_THROW("PARSE:Conv Data and Data Zero Point must have same type");
+
+            if(is_quant_conv)
+            {
+                ret = args[index];
+            }
+        }
+        else
+        {
+            if(is_quant_conv)
+            {
+                float symmetric_value = get_symmetric_value(ret);
+                ret                   = info.add_literal(migraphx::literal{
+                    migraphx::shape{input->get_shape().type(), {1}, {0}}, {symmetric_value}});
+            }
+        }
+
+        return ret;
+    }
+
+    static bool is_symmetric_zero_point(instruction_ref zp)
+    {
+        if(not zp->can_eval())
+            return false;
+
+        float symmetric_value = get_symmetric_value(zp);
+
+        bool all_zeros = false;
+        zp->eval().visit([&](auto z) {
+            all_zeros = std::all_of(
+                z.begin(), z.end(), [&](auto val) { return float_equal(val, symmetric_value); });
+        });
+        return all_zeros;
+    }
+
+    static auto
+    qparam_broadcast_op(instruction_ref qparam, std::vector<std::size_t> lens, std::size_t axis)
+    {
+        if(qparam->get_shape().scalar())
+        {
+            return migraphx::make_op("multibroadcast", {{"out_lens", lens}});
+        }
+        else
+        {
+            return migraphx::make_op("broadcast", {{"out_lens", lens}, {"axis", axis}});
+        }
+    }
+
+    static instruction_ref handle_quant_bias(const instruction_ref& input,
+                                             const instruction_ref& x,
+                                             const instruction_ref& weights,
+                                             const instruction_ref& x_zp,
+                                             const instruction_ref& w_zp,
+                                             onnx_parser::node_info& info)
+    {
+        instruction_ref ret = input;
+        if(not is_symmetric_zero_point(x_zp))
+        {
+            auto out_zp_1 = info.add_common_op("quant_convolution", x_zp, weights);
```

Comment:
I think these are still not using the attributes from the original quant conv. You should use `op` from L295

Reply:
Done